cpp/include/Ice/Communicator.h

```diff
@@ -362,18 +362,20 @@ namespace Ice
          */
         FacetMap findAllAdminFacets();
 
-#ifdef ICE_SWIFT
+#ifdef __APPLE__
         /**
-         * Returns the client dispatch queue.
+         * Returns the client dispatch queue, if any.
          * @return The dispatch queue associated wih this Communicator's
-         * client thread pool.
+         * client thread pool, or nullptr if none is configured.
+         * @remarks This operation is only available on Apple platforms.
```

Comment:
When we generate the doc, we'll define __APPLE__?

Reply:
We configure doxygen to expand all macros, see https://github.com/zeroc-ice/ice/blob/690eff4486d46ca6a970f0a39e3cd2fcacbe149a/cpp/doxygen/Doxyfile#L2316

This is also required fro the new SSL platform-specific APIs